apify-docs-theme/src/theme/custom.css

```diff
@@ -635,10 +889,27 @@ html[data-theme='dark'] .actionLink:hover::after {
     margin-left: 8px;
 }
 
+.cardContentWrapper {
+    display: flex;
+    padding-top: 25.6px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 25.6px;
```

Comment:
This doesn't seem right. If ti only affects these cards (viz screenshot) I'd go for 16px
![image](https://github.com/apify/apify-docs/assets/77627332/19e22bc4-ac54-4eed-b4c6-923ba9845477)